istioctl/cmd/dashboard.go

```diff
@@ -451,6 +458,9 @@ func dashboard() *cobra.Command {
 		"Address to listen on. Only accepts IP address or localhost as a value. "+
 			"When localhost is supplied, istioctl will try to bind on both 127.0.0.1 and ::1 "+
 			"and will fail if neither of these address are available to bind.")
+	dashboardCmd.PersistentFlags().BoolVar(&skipOpen, "skipOpen", false,
+		"When --skipOpen is supplied, istioctl dashboard will not open the browser. "+
+			"Default is false which means istioctl dashboard will always open a browser to view the dashboard.")
```

Comment:
It might make more sense to call the new CLI argument `--browser` and have the default `true`.  Thus, users who don't want one could say `--browser=false`.  "skipOpen" is awkward to document.